classes/Currency.php

```diff
@@ -863,13 +863,17 @@ public static function getIdByIsoCode($isoCode, $idShop = 0, $forceRefreshCache
             $query = Currency::getIdByQuery($idShop, $includeDeleted);
             $query->where('iso_code = \'' . pSQL($isoCode) . '\'');
 
-            $result = (int) Db::getInstance(_PS_USE_SQL_SLAVE_)->getValue($query->build());
+            $result = Db::getInstance(_PS_USE_SQL_SLAVE_)->getValue($query->build());
```

Comment:
This is weird.. I thought this change was from a PR made by @Progi1984 
How come it's in this PR, bad rebase?
(Same for Language modif)

Reply:
Initial PR https://github.com/PrestaShop/PrestaShop/pull/20398